source/tests/system/nisync_driver_api_tests.cpp

```diff
@@ -1177,32 +1197,32 @@ TEST_F(NiSyncDriver6683Test, SetTimeReference8021AS_ReturnsSuccess)
   auto grpcStatus = call_SetTimeReference8021AS(&viStatus);
 
   EXPECT_TRUE(grpcStatus.ok());
-  #if defined(_MSC_VER)
+#if defined(_MSC_VER)
   // SetTimeReference8021AS is only supported on Linux RT targets.
   EXPECT_EQ(NISYNC_ERROR_FEATURE_NOT_SUPPORTED, viStatus);
-  #else
+#else
   EXPECT_EQ(VI_SUCCESS, viStatus);
-  #endif
+#endif
 }
 
 TEST_F(NiSyncDriver6683Test, CreateClearFutureTimeEvent_ReturnsSuccess)
 {
   ViStatus viStatusCreate;
   ViUInt32 timeSeconds = GetCurrentBoardTimeSeconds() + 30;
   auto grpcStatusCreate = call_CreateFutureTimeEvent(
-     NISYNC_VAL_PFI1, // terminalName
-     NISYNC_VAL_LEVEL_LOW, // outputLevel
-     timeSeconds,
-     0, // timeNanoseconds, ignored
-     0, // timeFractionalNanoseconds, ignored
-     &viStatusCreate);
+      NISYNC_VAL_PFI1,       // terminalName
```

Comment:
I feel like we need to mass save the test directory so auto styling changes gets removed from our PRs

Reply:
True... I'll try to see about having a follow up PR with just a mass save.